Sem_4.py

```diff
@@ -0,0 +1,103 @@
+import sys
+import time
+import requests
+import multiprocessing
+import concurrent.futures
+import asyncio
+import aiohttp
+
+
+urls = [
+    'https://ru.freepik.com/popular-photos',
+    'https://ru.freepik.com/free-photo/peaceful-cute-horses-in-the-nature_17342857.htm',
+    'https:/https://ru.freepik.com/free-photo/front-view-woman-with-flowers-outdoors_34002136.htm'
+]
+
+
+def download_image(url):
+    response = requests.get(url, stream=True)
+    if response.status_code == 200:
+        filename = url.split('/')[-1]
+        with open(filename, 'wb') as file:
+            for chunk in response.iter_content(1024):
+                if chunk:
+                    file.write(chunk)
```

Comment:
А зачем так? Почему просто не 
```suggestion
            file.write(response.content)
```